server/engine/npc-generation/npcData.ts

```diff
@@ -6163,11 +6164,11 @@ export const npcData = {
         "I pocket anything I see that might have some value.",
       ],
       bond: [
-        'I fleeced the wrong person, a lord called <<print setup.npcData.raceTraits["human"].genderTraits["man"].firstName.seededrandom()>>, and must work to ensure that he never crosses paths with me or those I care about.',
-        "I owe everything to my mentor <<print setup.npcData.raceTraits['human'].genderTraits['man'].firstName.seededrandom()>>--a horrible person who's probably rotting in jail somewhere.",
-        "Somewhere out there I have a child, litte <<print setup.npcData.raceTraits['human'].genderTraits['man'].firstName.seededrandom()>>, who doesn't know me. I'm going to try and make the world better for him.",
+        'I fleeced the wrong person, a lord called <<print setup.npcData.raceTraits["human"].genderTraits["man"].randomValue(firstName)>>, and must work to ensure that he never crosses paths with me or those I care about.',
+        "I owe everything to my mentor <<print setup.npcData.raceTraits['human'].genderTraits['man'].randomValue(firstName)>>--a horrible person who's probably rotting in jail somewhere.",
+        "Somewhere out there I have a child, litte <<print setup.npcData.raceTraits['human'].genderTraits['man'].randomValue(firstName)>>, who doesn't know me. I'm going to try and make the world better for him.",
```

Comment:
`randomValue()` should wrap the whole setup...-part.
This applies for all three lines vice here, and the change below.